crates/h264/src/lib.rs

```diff
@@ -25,18 +25,18 @@
 //!
 //! use bytes::Bytes;
 //!
-//! use scuffle_h264::{Sps, AVCDecoderConfigurationRecord};
+//! use scuffle_h264::{AVCDecoderConfigurationRecord, Sps};
 //!
 //! // A sample h264 bytestream to parse
-//! let data = Bytes::from(b"\x01d\0\x1f\xff\xe1\0\x1dgd\0\x1f\xac\xd9A\xe0m\xf9\xe6\xa0  (\0\0\x03\0\x08\0\0\x03\x01\xe0x\xc1\x8c\xb0\x01\0\x06h\xeb\xe3\xcb\"\xc0\xfd\xf8\xf8\0".to_vec());
+//! let data = Bytes::from(b"\x01d\0\x1f\xff\xe1\0\x17\x67\x64\x00\x1F\xAC\xD9\x41\xE0\x6D\xF9\xE6\xA0\x20\x20\x28\x00\x00\x00\x08\x00\x00\x01\xE0\x01\0\x06h\xeb\xe3\xcb\"\xc0\xfd\xf8\xf8\0".to_vec());
```

Comment:
Maybe hide this in the docs since raw bytes dont make a lot of sense. Rename the variable to `bytes`  and then prefix the line with a `#` to disable it from being rendered by the docs.rs